docs/docs-components/COMPONENT_DATA.js

```diff
@@ -3358,6 +3359,58 @@ const BUILDING_BLOCKS_LIST: $ReadOnlyArray<ListItemType> = [
       figma: null,
     },
   },
+  {
+    svg: <SheetMobile />,
+    name: 'SheetMobile',
+    aliases: ['Panel', 'Dialog', 'Drawer', 'Tray'],
+    hasDarkBackground: false,
+    description:
+      'SheetMobile is a mobile only component. The SheetMobile is a supplementary container that sits on top of the screen’s primary content and can be dismissed in order to interact with the underlying content. Sheets can contain a wide variety of information and layouts, including menu items, actions, and supplemental content.',
```

Comment:
nit: no article needed in "The SheetMobile is a supplementary…"

Reply:
DONE

Reply:
DONE

> I'll keep reviewing this PR later, but can you pull the Dev Preview fixes for Modal, ModalAlert, and OverlayPanel out into their own PR? This PR is doing too much.

DONE